website_sale_category_description/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Jairo Llopis <jairo.llopis@tecnativa.com> (`Tecnativa <https://www.tecnativa.com/>`__)
```

Comment:
We use to put company and indented, the person (without the email for avoiding "spam").